src/MICore/CommandFactories/gdb.cs

```diff
@@ -183,6 +183,25 @@ public override async Task<List<ulong>> StartAddressesForLine(string file, uint
             return addresses;
         }
 
+        private async Task JumpInternal(string target)
+        {
+            // temporary breakpoint + jump
+            await _debugger.CmdAsync("-break-insert -t " + target, ResultClass.done);
```

Comment:
With this temporary breakpoint not registered, when we get a stopping event it will be unknown and returned as an exception.

https://github.com/microsoft/MIEngine/blob/728f403c563c2c50c46542e9b44271a7f8d5895b/src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs#L1179

I think we need to treat this as an async break.

Reply:
What do you mean by not registered?

Reply:
On stopping and when it is a breakpoint, we go through a list of breakpoints:
https://github.com/microsoft/MIEngine/blob/728f403c563c2c50c46542e9b44271a7f8d5895b/src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs#L1132

If if there are none, we assume it is an entrypoint breakpoint, embedded, we hit a bp pending deletion, or it is an exception.

Reply:
Indeed. It does receive a `breakpoint-modified` event and `disp` is `"del"` fwiw.
```sh
<-1067-break-insert -t *0x5555555573B1
->1067^done,bkpt={number="3",type="breakpoint",disp="del",enabled="y",addr="0x00005555555573b1",func="main(int, "...
<-1068-exec-jump *0x5555555573B1
->1068^running
->*running,thread-id="all"
->=breakpoint-modified,bkpt={number="3",type="breakpoint",disp="del",enabled="y",addr="0x00005555555573b1",func="main(int, "...
->*stopped,reason="breakpoint-hit",disp="del",bkptno="3",frame={addr="0x00005555555573b1",func="main",args=[{name="argc",value="1"},{name="argv",value="0x7fffffffdb68"}],file="../test.cpp",fullname="/tmp/test.cpp",line="28",arch="i386:x86-64"},thread-id="1",stopped-threads="all",core="6"
->=breakpoint-deleted,id="3"

```